res/values-zh-rHK/strings.xml

```diff
@@ -16,33 +16,33 @@
 
```

Comment:
All PRs should be submitted as Gerrit Changes. Please see the documentation at https://wiki.cyanogenmod.org/w/Doc:_using_gerrit